### PR TITLE
Feature/fc 109

### PR DESCRIPTION
**Pull request by tjsummour17**

### CODE REVIEW

File: lib/utilities/database.dart [hunk 1]
```diff
@@ -176,23 +173,20 @@ class Database {
...
-    final FacebookLoginResult result =
-    await facebookSignIn.logIn(['email']);
+    final FacebookLoginResult result = await facebookSignIn.logIn(['email']);
     switch (result.status) {
       case FacebookLoginStatus.loggedIn:
         final token = result.accessToken.token;
-        final graphResponse = await http.get(
-            'https://graph.facebook.com/v2.12/me?fields=name,first_name,last_name,email&access_token=${token}');
+        final graphResponse = await http
+            .get('https://graph.facebook.com/v2.12/me?fields=name,first_name,last_name,email&access_token=${token}');
```
Comment (osama-al-basha) [inline lib/utilities/database.dart:181]: @TalalSammour This link should be in a constants class and any other hardcoded strings in the app.
The point is to store them in one file and apply changes later to only one place.
Please loop over any hardcoded string and put them into one file.

File: lib/utilities/database.dart [hunk 2]
```diff
@@ -210,44 +204,35 @@ class Database {
...
       }
     } catch (e) {
       print('${e.toString()}');
       return new AppUser.User();
     }
   }
 
-  static Future<AudioPlayer> _playLocalAssetOnSend() async =>
-      await AudioCache().play("newMessage.mp3");
+  static Future<AudioPlayer> _playLocalAssetOnSend() async => await AudioCache().play("newMessage.mp3");
```
Comment (osama-al-basha) [inline lib/utilities/database.dart:227]: Same thing here for the file name and extension.
@TalalSammour

File: lib/utilities/database.dart [hunk 3]
```diff
@@ -210,44 +204,35 @@ class Database {
...
 
-  static Future<AudioPlayer> _playLocalAssetOnJoinChatRoom() async =>
-      await AudioCache().play("joinChat.mp3");
+  static Future<AudioPlayer> _playLocalAssetOnJoinChatRoom() async => await AudioCache().play("joinChat.mp3");
```
Comment (osama-al-basha) [inline lib/utilities/database.dart:229]: Here as well.

File: lib/utilities/database.dart [hunk 4]
```diff
@@ -210,44 +204,35 @@ class Database {
...
 
-  static Future<AudioPlayer> _playLocalAssetOnLeaveChatRoom() async =>
-      await AudioCache().play("leaveChat.mp3");
+  static Future<AudioPlayer> _playLocalAssetOnLeaveChatRoom() async => await AudioCache().play("leaveChat.mp3");
```
Comment (osama-al-basha) [inline lib/utilities/database.dart:231]: Same thing

File: lib/utilities/database.dart (hunk 1 above)
Comment (tjsummour17) [reply lib/utilities/database.dart:181]: @osama-al-basha 
but this link has dynamic variable $token as you can see

File: lib/utilities/database.dart (hunk 2 above)
Comment (tjsummour17) [reply lib/utilities/database.dart:227]: ok no problem

File: lib/utilities/database.dart (hunk 1 above)
Comment (osama-al-basha) [reply lib/utilities/database.dart:181]: @TalalSammour Yes, I mean the whole link not only the "token"

**Review by osama-al-basha — APPROVED** [review]

**Pull request merged**